manifests/m/MHNexus/HxD/2.5.0.0/MHNexus.HxD.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=MDSU.CRLF.7-3-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MHNexus.HxD
+PackageVersion: 2.5.0.0
```

```suggestion
PackageVersion: 2.5
```
Should match Control Panel